pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -119,6 +121,14 @@ func (c *cephStatusChecker) checkStatus() {
 	}
 
 	logger.Debugf("cluster status: %+v", status)
+
+	if status.Health.Status != "HEALTH_OK" {
```

Comment:
Let's move this block after the call to `updateCephStatus()` so the status update on the CR can be completed first.

Reply:
Done